torch/csrc/jit/codegen/cuda/lower_expr_sort.cpp

```diff
@@ -621,16 +661,10 @@ std::unordered_set<ExprGroupConnections*> ExprSegmentationSorter::
   return removed_edges;
 }
 
-// TODO: This function may be sub optimial. If we find that an iteration domain
-// matches later in the other domain, we will hold all other iteration domains
-// until that one matches. There may be cases where duplicating that iteration
-// domain, and moving on could be more efficient.
-ExprGroup* ExprSegmentationSorter::makeMergedNode(
-    ExprGroup* sg1,
-    ExprGroup* sg2) {
-  std::vector<IterDomain*> resulting_ca_axes;
-  auto& domain1 = sg1->payload()->ca_domains_;
-  auto& domain2 = sg2->payload()->ca_domains_;
+std::vector<IterDomain*> mergeDomains(
```

Comment:
nit: Should this be placed in any anonymous namespace?

Reply:
Yes, it should be.